src/components/Transactions.tsx

```diff
@@ -34,6 +35,7 @@ const Transactions = ({ type, ...transactionDisplayProps }: Props) => {
   const { transactions } = useTransactionContext();
   const { dispatchTransaction } = useUpdateTransactionContext();
   const { dispatchMessage } = useUpdateMessageContext();
+  useResetTransactionState(type);
```

Comment:
Can you elaborate what this is about? What is `type` and why if we set it back to a previous value it should reset the transaction state?

Reply:
Of course ... basically when i switch actions tabs ... let's say from "Transfer" to "CustomCall" the payload and estimatedFee needs to be reseted otherwise the tabs would show the values calculated in the previous tab.

Anyway it is not working 100% as expected since for this specific scenario i have an issue:

1) Fill the form for a Transaction tab.
2) Switch action to Custom Call

Estimated Fee remains the same as the one calculated from the previous tab. This is only happen for this scenario, the other switch do not face this issue. As i considered that this issue was out of the scope of this PR i have logged the following issues in order to track this:

https://github.com/paritytech/parity-bridges-ui/issues/195
https://github.com/paritytech/parity-bridges-ui/issues/209
https://github.com/paritytech/parity-bridges-ui/issues/207